test/bin/scenario.sh

```diff
@@ -1087,6 +1087,13 @@ run_gingko_tests() {
     local vmname="${1}"
     shift
 
+    # Parse optional filter argument (can be ~Disruptive or Multus)
+    local filter=""
```

Comment:
We could probably assume that "$@" is the filter here, right? We do not have any other positional arguments, do we?